Scripts/hello_world_deborah_camila.js

```diff
@@ -0,0 +1,5 @@
+// LANGUAGE: Javascript
+// AUTHOR: Deborah Camila
+// GITHUB: https://github.com/debcamila
+
+console.log('Hello World!');
```

Comment:
Dica: sempre deixar uma linha em branco ao fim de um arquivo de código para evitar este warning: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file